ansible/roles/ml-analytics-adhoc-jobs/tasks/main.yaml

```diff
@@ -104,3 +104,31 @@
     var: out.stdout_lines
   tags:
     - project-ingest
+
+- name: Remove existing input file
+  file: path="{{ BASEPATH }}/input.xlsx" state=absent
+  tags:
+    - program-script
+
+- name: Copy Input.xlsx
+  become: yes
+  become_user: data-pipeline
+  copy: src=/tmp/input.xlsx dest="{{ BASEPATH }}/input.xlsx" force=true
```

Comment:
As you are referencing the /tmp folder, how we are sure that this file is always present here? How the input.xlsx file is copied to /tmp ? This task will fail if the file is not available